### PR TITLE
Features/queries mutation

### LINKED ISSUE #27: Mutation to a match
As an Athlete I should be able to send a mutation to the server when I have a match with a Team

### LINKED ISSUE #35: Register a new athlete mutation
As a new user I should be able to register me as a new athlete sending a mutation to the server

### LINKED ISSUE #37: Match a team button error hadling
As an Athlete I should be able to see a message in the button when I send a match to a team.

### LINKED ISSUE #39: Disabled match button
As an Athlete I should not be able to add a twice team in my interested section, disabling the match button functionality.

### LINKED ISSUE #40: Search view with all teams
As an Athlete, I should be able to see a carousel with all the available teams in the app.

### PR DESCRIPTION
**Pull request by Andrey-1992**
### Type of change:
  - [x] New feature  
  - [ ] Bug Fix  

#### What does this PR do?
- Incorporate mutations for match and create a new athlete
- Styling and clean code in AthleteSearch component

#### How should this be tested?
- [x] npm start
- [ ] Cypress
- [ ] RSpec

#### Any background context you want to provide?
- The data displayed is from the queries

### Check the correct boxes:
  - [x] The code will run locally  
  - [x] This broke nothing  
  - [ ] This broke some stuff  
  - [ ] This broke everything  

### Checklist:
  - [x] My code has no unused/commented out code  
  - [x] I have reviewed my code  
  - [x] I have commented my code, particularly in hard-to-understand areas  
  - [ ] I have fully tested my code

### CODE REVIEW

File: src/GraphQL/Mutations.js [hunk 1]
```diff
@@ -1,9 +1,80 @@
...
+      talent {
+        id,
+        name
+      }
+    }
+  }
+`
+
+
+// talent {
```
Comment (williamphelps13) [inline src/GraphQL/Mutations.js:62]: Remove commented out code?

File: src/components/Athlete/Athlete.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ import "react-responsive-carousel/lib/styles/carousel.min.css";
 const Athlete = ({ data }) => {
   const [windowWidth, setWindowWidth] = useState(0)
   const [mobile, setMobile] = useState(false)
```
Comment (williamphelps13) [inline src/components/Athlete/Athlete.js:18]: Super interesting that the carousel stores these in state

File: src/components/AthleteSearch/AthleteSearch.css [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.AthleteSearch {
```
Comment (williamphelps13) [inline src/components/AthleteSearch/AthleteSearch.css:1]: Thanks Andrey!

File: src/components/AthleteSearch/AthleteSearch.css [hunk 4]
```diff
@@ -55,17 +57,18 @@
...
+  padding-right: 2%;
 }
 
 .team-name-search {
-  margin: 2%;
+  /* margin: 2%; */
   background: whitesmoke;
   color: #c6c6c6;
   font-size: 18px;
   /* padding: 10px 10px 10px 5px; */
```
Comment (williamphelps13) [inline src/components/AthleteSearch/AthleteSearch.css:68]: Remove commented out

File: src/components/AthleteSearch/AthleteSearch.css [hunk 5]
```diff
@@ -55,17 +57,18 @@
...
   border: none;
   border: 3px solid rgb(108, 3, 3);
-  margin-top: 2%;
+  /* margin-top: 1%; */
```
Comment (williamphelps13) [inline src/components/AthleteSearch/AthleteSearch.css:71]: Remove commented out?

File: src/components/AthleteSearch/AthleteSearch.js [hunk 6]
```diff
@@ -5,141 +5,130 @@ import Header from '../Header/Header';
...
-      event.preventDefault();
-      setSearchStatus(false)
-      getData(); // Function to have a location radius
-      findTeams(); // Function to filter teams based on the user text filter
-      clearInputs();
-    }
+  const getFormInfo = (event) => {
+    event.preventDefault();
+    setSearchStatus(false);
+    // getData();
```
Comment (williamphelps13) [inline src/components/AthleteSearch/AthleteSearch.js:50]: Remove comment?

File: src/components/AthleteSearch/AthleteSearch.js [hunk 7]
```diff
@@ -5,141 +5,130 @@ import Header from '../Header/Header';
...
+import { LOAD_ALL_CLUBS } from '../../GraphQL/Queries';
 import { CREATE_MATCH } from '../../GraphQL/Mutations';
 import { useMutation } from '@apollo/client';
 
-
 const AthleteSearch = ({userData}) => {
   const {id} = useParams();
   const [allTeams, setAllTeams] = useState({});
   const [filteredTeams, setFilteredTeams] = useState([]);
   const [teamName, setTeamName] = useState("");
```
Comment (williamphelps13) [inline src/components/AthleteSearch/AthleteSearch.js:16]: Could change to single quote for consistency?

File: src/components/AthleteSearch/AthleteSearch.js [hunk 8]
```diff
@@ -5,141 +5,130 @@ import Header from '../Header/Header';
...
-      // console.log('teamName:', teamName)
-      // console.log('filteredTeams:',filteredTeams)
-
-      // const teams = allTeams.allClubs;
-      // const teamByName = teams.filter(team => team.name.includes(teamName) || teamName === 'All')
-      // console.log(teamByName)
-      // const teamsByRadius = zipCodeRadius;
-      // console.log(zipCodeRadius)
-    }
+    console.log(zipCodeRadius.dataList)
```
Comment (williamphelps13) [inline src/components/AthleteSearch/AthleteSearch.js:62]: Remove console.log?

File: src/components/AthleteSearch/AthleteSearch.js [hunk 9]
```diff
@@ -5,141 +5,130 @@ import Header from '../Header/Header';
...
+  console.log('teamLocation:', typeof teamLocation)
 
-    // fetch(`https://api.zip-codes.com/ZipCodesAPI.svc/1.0/FindZipCodesInRadius?zipcode=${userZipCode.zipcode}&maximumradius=${teamLocation}&minimumradius=0&country=US&key=${apiKey}`)
-    //   .then(res => res.json())
-    //   .then(data => setZipCodeRadius(data))
-      // .then(console.log(zipCodeRadius))
-      // .then(data => console.log('ZipCodeData: ', data));
+  const teams = allTeams.allClubs;
+  const filterInputs = setFilteredTeams(teams.filter(team => team.name.includes(teamName) || teamName === 'All'))
+  // console.log(filteredTeams)
```
Comment (williamphelps13) [inline src/components/AthleteSearch/AthleteSearch.js:74]: Remove comment?

File: src/components/RegistrationAthlete/RegistrationAthlete.js [hunk 10]
```diff
@@ -1,30 +1,84 @@
...
-    fortyYard: 0.00,
-    personJugRec: 0,
-    talents: [],
-    awards: '',
+    goalsMadeLs: 0,
+    verticalJump: 0,
+    forthyDash: 0,
+    jugglingRecord: 0,
+    talents: '',
+    awards: 'best gooly'
```
Comment (williamphelps13) [inline src/components/RegistrationAthlete/RegistrationAthlete.js:30]: Remove string?

File: src/components/RegistrationAthlete/RegistrationAthlete.js [hunk 11]
```diff
@@ -1,30 +1,84 @@
...
   });
 
+  const [createTalent, {error}] = useMutation(CREATE_TALENT);
+  
+  const registerTalent = () => {
+    // console.log(registrationAthlete)
```
Comment (williamphelps13) [inline src/components/RegistrationAthlete/RegistrationAthlete.js:36]: Remove console.log

File: src/components/RegistrationAthlete/RegistrationAthlete.js [hunk 12]
```diff
@@ -1,30 +1,84 @@
...
+        dominantFoot: registrationAthlete.dominantFoot,
+        goalsMadeLs: registrationAthlete.goalsMadeLs,
+        verticalJump: registrationAthlete.verticalJump,
+        fortyDash: registrationAthlete.fortyDash,
+        jugglingRecord: registrationAthlete.jugglingRecord,
+        talents: registrationAthlete.talents,
+        awards: registrationAthlete.awards
+      }
+    })
+    createTalent({
```
Comment (williamphelps13) [inline src/components/RegistrationAthlete/RegistrationAthlete.js:57]: I'll remove this, correct?

**Review by williamphelps13 — APPROVED** [review]
Excellent job on this branch! Super happy to have mutations working! I left some comments for cleaning up commented out code and console logs, but great job already with fixing CSS classes

File: src/GraphQL/Mutations.js [hunk 13]
```diff
@@ -1,9 +1,80 @@
...
+      match {
+        sportClubId,
+        talentId
+      }
+      errors
     }
   }
-`
\ No newline at end of file
+`
```
Comment (deebot10) [inline src/GraphQL/Mutations.js:13]: Nice work on this side.

File: src/components/RegistrationAthlete/RegistrationAthlete.js [hunk 14]
```diff
@@ -1,30 +1,84 @@
...
+    jugglingRecord: 0,
+    talents: '',
+    awards: 'best gooly'
   });
 
+  const [createTalent, {error}] = useMutation(CREATE_TALENT);
+  
+  const registerTalent = () => {
+    // console.log(registrationAthlete)
+    createTalent({
```
Comment (deebot10) [inline src/components/RegistrationAthlete/RegistrationAthlete.js:37]: Cool job

**Review by deebot10 — APPROVED** [review]

**Pull request merged**